mono_repo/lib/src/package_config.dart

```diff
@@ -186,6 +186,28 @@ class CIJob implements HasStageName {
   /// The description of the job in the CI environment.
   String get name => description ?? _taskCommandsTickQuoted.join(', ');
 
+  String get travisSdk {
+    // Map from Travis Dart SDK lingo to setup-dart-action
+    for (var entry in _travisToGitHubOsMap.entries) {
```

Comment:
Not sure the map is worth it here, given its just one entry and I somewhat doubt we will have more of these. It would be a lot easier to read if it just checked for `edge` specifically here.

Reply:
I kinda like having a single-point-of-truth here. 🤷